draft-ietf-tls-tls13.md

```diff
@@ -3566,7 +3563,7 @@ challenge
   not trailing) zero bytes to make it 32 bytes long.
 {:br }
 
-Note: Requests to resume a TLS session MUST use a TLS ClientHello.
+Note: Requests to resume a TLS session MUST use a TLS client hello.
 
```

Comment:
I think it should stay as ClientHello/ServerHello where applicable, and only switch to "hello message" when it covers multiple. I think it's easier to keep track of things if the struct names are used explicitly when reference. Also makes searching the document for stuff about them a little easier.